sphinx/search/ru.py

```diff
@@ -52,7 +52,7 @@
 от             | away from
 меня           | me
 еще            | still, yet, more
-нет            | no, there isnt/arent
+нет            | no, there isn't/aren't
```

Comment:
I don't know this change is really correct. This is a database of Russian search engine. Please ensure the correctness of this (or revert).

It seems this list is copied from http://snowball.tartarus.org/algorithms/russian/stop.txt. So we have to update it first if something wrong.

Reply:
If it depends on the outside of the sphinx, it might be safe to leave it as it is. If you don't mind, I will revert it.